### PR TITLE
why become a teacher new page

### PR DESCRIPTION
**Pull request by emmaachesongray**
### Trello card
https://trello.com/c/MA9IVFdL/7037

### Context
We identified a content gap on GIT. So we need to add more content about why users should become a teacher.

### Changes proposed in this pull request
This PR adds a new page of content to the life as a teacher section to meet this content gap.

### CODE REVIEW

**Review by gemmadallmandfe — COMMENTED** [review]
@emmaachesongray This looks great - have left a few comments

Let's discuss where this content sits with @Sarah-DfE on Thursday!

File: app/views/content/life-as-a-teacher/teaching-as-a-career/_categories.html.erb [hunk 1]
```diff
@@ -3,6 +3,12 @@
     <section class="container category__cards main-section">
       <nav class="category__nav-cards">
         <ul>
+          <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Why become a teacher",
+              description: "Explore the many reasons why you should become a teacher in England. From making a difference to exploring your creativity, holidays, pay, and much more.",
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/teaching-as-a-career/_categories.html.erb:9]: Not sure about this, it sounds a bit like they're exploring their creativity, holidays, pay  etc

Maybe:
From making a difference from day 1, to exploring your creativity and benefitting from a competitive salary and generous holidays.

Might be a bit wordy now though!

File: app/views/content/life-as-a-teacher/teaching-as-a-career/why-teach.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: Why become a teacher
+description: |- 
+  Explore the many reasons why you should become a teacher in England. From making a difference to exploring your creativity, holidays, pay, and much more.
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/teaching-as-a-career/why-teach.md:4]: As above

File: app/views/content/life-as-a-teacher/teaching-as-a-career/why-teach/_article.html.erb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  <section class="col col-720 col-space-l-top">
+    <p>There’s no career quite like teaching. From sharing your passion for a subject, to having a job where every day is different, there’s lots of reasons why people choose to become teachers in England.</p>
+
+    <h2 class="heading--box-blue">Make a difference</h2>
+
+    <p>From day one as a teacher, you’ll be empowered to make decisions and have an impact. Every lesson you teach or conversation you have in a school corridor has the opportunity to inspire your pupils, shape their futures, and the world around them.</p>
+
+    <p><%= render 'content/shared/quotes/quote_why_teach_chelsea' %></p>
+
+    <p>Your impact won’t be limited to your pupils. You’ll have opportunities to make a difference to your school, colleagues, and local community.</p>
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/teaching-as-a-career/why-teach/_article.html.erb:11]:
```suggestion
    <p>Your impact will not be limited to your pupils. You’ll have opportunities to make a difference to your school, colleagues, and local community.</p>
```

File: app/views/content/life-as-a-teacher/teaching-as-a-career/why-teach/_article.html.erb [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+    <h2 class="heading--box-blue">Explore your creativity</h2>
+
+    <p>As a teacher, you’ll have the freedom and trust to use your knowledge, passion and creativity to teach curriculums in your own way. From themed learning days and engaging activities to creative wall displays, you can bring your personality and passion to the classroom from the very first day.</p>
+
+    <p><%= render 'content/shared/quotes/quote_why_teach_helen' %></p>
+
+    <h2 class="heading--box-blue">Use skills from your degree or previous career</h2>
+
+    <p>Teaching provides you with a great environment to use the skills and knowledge you’ve developed during a degree or a previous career. You’ll share your passion with your pupils and continue to learn as you teach.</p>
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/teaching-as-a-career/why-teach/_article.html.erb:23]: Worth a link to careers changers section here?

**Review by Sarah-DfE — COMMENTED** [review]
Love it! Just added a few small suggestions

File: app/views/content/life-as-a-teacher/teaching-as-a-career/_categories.html.erb (hunk 1 above)
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/teaching-as-a-career/_categories.html.erb:9]:
```suggestion
              description: "Explore the many reasons why you should become a teacher in England. From making a difference from day 1, to exploring your creativity and benefitting from a competitive salary and generous holidays.",
```

File: app/views/content/life-as-a-teacher/teaching-as-a-career/why-teach.md (hunk 2 above)
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/teaching-as-a-career/why-teach.md:4]:
```suggestion
  Explore the many reasons why you should become a teacher in England. From making a difference from day 1, to exploring your creativity and benefitting from a competitive salary and generous holidays.
```

File: app/views/content/life-as-a-teacher/teaching-as-a-career/why-teach.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+title: Why become a teacher
+description: |- 
+  Explore the many reasons why you should become a teacher in England. From making a difference from day 1, to exploring your creativity and benefitting from a competitive salary and generous holidays.
```
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/teaching-as-a-career/why-teach.md:4]:
```suggestion
  Explore the many reasons why you should become a teacher in England. From making a difference from day one, to exploring your creativity and benefitting from a competitive salary and generous holidays.
```

File: app/views/content/life-as-a-teacher/change-careers/benefits-of-changing-careers/_article.html.erb [hunk 6]
```diff
@@ -8,6 +8,7 @@
     <p>The skills and experiences you bring from previous careers are invaluable in the classroom. From communication and public speaking, to problem solving and people management, your pupils will learn more from you than you might think.</p>
     <p>As a teacher, you can watch pupils progress as they learn, making an impact from your very first lesson.</p>
     <p><%= render 'content/shared/quotes/quote_career_change_jodie' %></p>
+    <p><a href="/life-as-a-teacher/teaching-as-a-career/why-teach">Find out more about why other people choose to become a teacher</a>.</p>
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/change-careers/benefits-of-changing-careers/_article.html.erb:11]:
```suggestion
    <p><a href="/life-as-a-teacher/teaching-as-a-career/why-teach">Find out more about why other people choose to become teachers</a>.</p>
```

**Review by Sarah-DfE — COMMENTED** [review]
Just one tiny tweak then this is ready to go

**Review by Sarah-DfE — APPROVED** [review]
Looks good to me!
